src/rules/export.js

```diff
@@ -34,7 +35,14 @@ module.exports = {
         if (node.declaration == null) return
 
         if (node.declaration.id != null) {
-          addNamed(node.declaration.id.name, node.declaration.id)
+          if ([
+            'TSTypeAliasDeclaration',
+            'TSInterfaceDeclaration',
+          ].includes(node.declaration.type)) {
```

Comment:
`includes` isn't available on node 4; we'll need to add the `array-includes` package here.